k8s/on-prem/openshift-installer/create_openshift_cluster.yml

```diff
@@ -0,0 +1,147 @@
+---
+- hosts: localhost
+
+  vars_files:
+    - vars.yml
+  
+  vars:
+    vpc_name:
```

Comment:
@shashank855 what is the vpc_name here? is anywhere your are using it?

Reply:
@nsathyaseelan I made the change.